### PR TITLE
Simplify Logic, Fixes #52

### LINKED ISSUE #52: Simplify Logic
Hi there!  I looked through the code and found that it can be a little simplified. The snippet below can be replaced by a simple [`process.versions.node`]( https://nodejs.org/api/process.html#process_process_versions):

https://github.com/srod/node-version/blob/4efd69fbe2bf97907a9216937fac82fe81947415/index.js#L8-L12

```js
var version = process.versions.node;
```

And this can be replaced with `long: version`:

https://github.com/srod/node-version/blob/4efd69fbe2bf97907a9216937fac82fe81947415/index.js#L19

If you don't mind, I can open a PR.

Thanks, Madhav.

### PR DESCRIPTION
**Pull request by madhavarshney**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,18 +5,14 @@
...
-  if (version.indexOf('v') > -1) {
-    version = version.split('v')[1];
-  }
+  var version = process.versions.node;
 
   var split = version.split('.');
 
   return {
-    original: process.version,
+    original: 'v' + version,
```
Comment (madhavarshney) [inline index.js]: Is this an over-optimization?

Comment (srod) [reply index.js]: or maybe keep `process.version` here ?

**Comment by srod** [comment]
I will check that tomorrow :)

Thanks

File: index.js (hunk 1 above)
Comment (madhavarshney) [reply index.js]: Your call, if you want, I can revert that.

**Comment by srod** [comment]
Merged, thanks!

**Pull request merged**